scripts/copy_usd_profits.sh

```diff
@@ -0,0 +1,6 @@
+mevInspectPoolId=$(kubectl get pods | sed -n -e '/^mev-inspect-/p' | sed '/^mev-inspect-workers/d' | awk '{print $1}')
```

Comment:
could be good to keep that script:
- [ ] add a comment to say what this is doing
- [ ] add #!/bin/bash